### DISCUSSION TITLE
SSD1803 not working on DOGS164

### DISCUSSION THREAD

**Discussion by sebastiangepp** [sstaub/SSD1803A_I2C#1] [Q&A]
Hi
I tried to implement the lib "ssd1803A_i2c" on my display and a esp8266 Wemos D1 mini, but I do not get it to respond. I also think the I2C is not working correctly since I am not getting an answer in the scanner.
Any help is much appriciated.

**Comment by sstaub** [comment]
Have you checked the wiring is correct like in the data sheet. If the I2C address scanner gives no answer I can't help because I have no ESP boards.

**Comment by sebastiangepp** [comment]
HiI checked the wiring again and found it. The pullup on reset was missing. I had it directly connected to the mcu. Now it works and i will play with it. Thanks
-------- Ursprüngliche Nachricht --------Von: Stefan Staub ***@***.***> Datum: 16.01.23  20:58  (GMT+01:00) An: sstaub/SSD1803A_I2C ***@***.***> Cc: sepp ***@***.***>, Author ***@***.***> Betreff: Re: [sstaub/SSD1803A_I2C] SSD1803 not working on DOGS164 (Discussion #1) 
Have you checked the wiring is correct like in the data sheet. If the I2C address scanner gives no answer I can't help because I have no ESP boards.
